l2-contracts/contracts/bridge/L2AssetRouter.sol

```diff
@@ -29,16 +29,17 @@ contract L2AssetRouter is IL2AssetRouter, Initializable {
     /// @dev The address of the L2 legacy shared bridge.
     address public immutable L2_LEGACY_SHARED_BRIDGE;
 
-    /// @dev The address of the L1 asset router counterpart.
-    address public override l1AssetRouter;
```

Comment:
Why rename it? The L2AssetRouter is not the L2SharedBrideLegacy, it will be at a different address, we don't need storage and interface backward compatibility. Same question for some of the functions, e.g. withdraw